assets/android/res/drawable/ic_compound_presence_outline_8_x_8.xml

```diff
@@ -1,12 +1,12 @@
 <vector xmlns:android="http://schemas.android.com/apk/res/android"
     android:width="8dp"
     android:height="8dp"
-    android:viewportWidth="8"
-    android:viewportHeight="8">
+    android:viewportWidth="24"
+    android:viewportHeight="24">
```

Comment:
Also, I'm unsure how important this is on Android, for the 8x8 icons.

Reply:
Thanks for catching, I've fixed this by bringing back the `viewBox` SVG attribute on icons.